tests/test_shard_sync_op.py

```diff
@@ -20,9 +20,6 @@
 
 def init_model_and_data(local_rank):
     model = nn.Linear(10, 10).cuda(local_rank)
-    # Make sure all devices have the same model.
-    dist.broadcast(model.weight.data, src=0)
-    dist.broadcast(model.bias.data, src=0)
```

Comment:
Why this is no longer needed?

Reply:
Because I disabled pytest randomly plugin.